client/modules/core/__tests__/main.js

```diff
@@ -0,0 +1,83 @@
+import { Meteor } from "meteor/meteor";
+
+import main from "../main";
+
+describe("Client/API/Core", () => {
```

Comment:
Change all `it` to `test`.

It also would be better to move `shopDomain`, `getDomain`, and `absoluteUrl` into their own file (maybe `domain.js`. You can export an object with just those three props and in `main.js`, import it and spread it onto the main object. This will help isolate better and keep dependencies to a minimum, which should help the non-Meteor unit tests pass more easily.

We also use the `.test.js` convention rather than `__tests__` folder. So you should have:

```
domain.js
domain.test.js
```

Reply:
sounds good